src/activities/content/ContentEntity.js

```diff
@@ -12,15 +13,17 @@ export const CONTENT_TYPES = {
 export class ContentEntity extends Entity {
 
 	/**
-	 * @returns {string} content-module link
+	 * @returns {string} content type
 	 */
 	getEntityType() {
 		// TODO - add more specific entity types to the activity-usage class array
-		// once we start working on non-module content types
+		// as we start working on new content types
 		if (!this._entity) {
 			return null;
 		} else if (this._entity.hasClass(CONTENT_TYPES.module)) {
 			return CONTENT_TYPES.module;
+		} else if (this._entity.hasClass(CONTENT_TYPES.weblink)) {
```

Comment:
Just a note that currently the `activity-usage` is not returning a `weblink` class on it. I'm not sure if we want/need this or if the existence of the `content-weblink` Link is enough. @alamarre @ctwomey1 what you do you think? Should an `activity-usage` for a given activity have a class that specifies the type of the activity in a meaningful way?

Reply:
This is currently the `activity-usage` response for a weblink activity:
![image](https://user-images.githubusercontent.com/2243995/103030164-d33a1080-4520-11eb-97ad-a833fb4e2ca1.png)

I left the `topic` class on it because I don't want to break any existing consumers of these `activity-usage` entities. I did not specifically add `weblink` but I certainly can if that is desired.

Reply:
Good to know! I agree, I can just replace it by checking for the existence of the `content-weblink` link.

Reply:
RE 
> Just a note that currently the activity-usage is not returning a weblink class on it. I'm not sure if we want/need this or if the existence of the content-weblink Link is enough. @alamarre @ctwomey1 what you do you think? Should an activity-usage for a given activity have a class that specifies the type of the activity in a meaningful way?

I think describing the entity fully would certainly make it more flexible for consumption.  It would all you to do things like pull all activities out of a collection with class "weblink" easier, but if current workflows don't require it, i wouldn't worry about it

Reply:
It seems like it's maybe not immediately necessary but I agree that it probably would be better to include it. I'll look to add a `weblink` class as an independent PR sometime in the near future.